src/qt/forms/modaloverlay.ui

```diff
@@ -334,6 +354,33 @@ QLabel { color: rgb(40,40,40);  }</string>
             <number>10</number>
            </property>
            <item>
```

Comment:
Nit, can't comment there, but remove `leftMargin` above to be consistent.

Reply:
Why remove?